pkg/nl/manager.go

```diff
@@ -26,9 +26,14 @@ const (
 var macPrefix = []byte("\x02\x54")
 
 type NetlinkManager struct {
```

Comment:
even though this isn't part of the MR, this should be called `Manager` since the package is already called netlink (well, `nl`).
https://go.dev/doc/effective_go#names

Reply:
Done